underfs/local/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,17 @@
+tachyon.root.logger=INFO, TEST_LOGGER
+tachyon.log.dir=.
+tachyon.log.file=logs/tests.log
```

Comment:
Could we change the logging dir to be consistent with the other log4j test properties? I think the idea is to have mvn clean cleanup the test logs as well.